### PR TITLE
Set up MkDocs

### PR DESCRIPTION
**Pull request by HEM2058**
MkDocs has been configured, and the "About Us" information has been extracted from the OSGeo Nepal wiki page.

### CODE REVIEW

**Review by iamtekson — COMMENTED** [review]
It is a good starting point. Thanks for creating this PR @HEM2058 vai.

**Comment by kshitijrajsharma** [comment]
@HEM2058  Can you also provide github.io for testing from your local fork ?

**Comment by HEM2058** [comment]
@kshitijrajsharma https://hem2058.github.io/RollingConvo/

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+site_name: osgeonepal
+site_url: https://hem2058.github.io/
```
Comment (iamtekson) [inline mkdocs.yml:2]: This one need to be changed to osgeonepal.github.io

**Review by iamtekson — COMMENTED** [review]
Overall, the documentation structure looks good to me. but looks like the `site/../` is the autogenerated file from mkdocs. Do we need to keep that in order to generate the site? I have done similar things using readthedocs & `sphinx` (here is the example: [geotile documentation](https://github.com/osgeonepal/geotile/tree/main/docs)). 

I think removing auto-generated files makes the repository clean. So, I suggest to remove those.

File: mkdocs.yml (hunk 1 above)
Comment (HEM2058) [reply mkdocs.yml:2]: Okay dai, I will try to deploy on osgeonepal page also.

**Comment by kshitijrajsharma** [comment]
Thank you for @HEM2058  for this PR . Excellent work ! Cheers ! 

Some Suggestion and Enhancements from my side  : 

- I have added auto deployment of docs through [github actions ](https://github.com/osgeonepal/rollingdoc/blob/master/.github/workflows/publish_mkdocs.yml)
- I have also installed several extensions such as dynamic navigation based on docs change , fonts , github linking , search bar etc ! Checkout [here](https://github.com/osgeonepal/rollingdoc/blob/master/mkdocs.yml)
- I have disabled readthe docs format and migrated to this new layout for easecase !

**Pull request merged**